config/main.py

```diff
@@ -313,9 +314,16 @@ def _restart_services():
         'lldp',
         'hostcfgd',
     ]
+
+    # We first run "systemctl reset-failed" to remove the "failed"
+    # status from all services before we attempt to restart them
+    click.echo("Resetting all failed services ...")
+    run_command("systemctl reset-failed")
```

Comment:
The concern I have is that this could possibly reset more services than the ones on the list we are meant to handle. Not sure if this is correct or desirable.

Reply:
so maybe change approach to reset counter per service ?

Reply:
Yes I believe that approach would be better.